pages/ecosystem.vue

```diff
@@ -168,24 +146,35 @@ useSeoMeta({
   twitterDescription: description,
 });
 
+const joinSectionCards = [
+  {
+    name: "Main tier",
+    description: "The main Qiskit packages maintained by IBM Quantum.",
+    icon: "quantum.svg",
+  },
+  {
+    name: "Extensions tier",
+    description: "IBM Quantum supported Qiskit extensions.",
+    icon: "transform--02.svg",
+  },
+  {
+    name: "Community tier",
+    description:
+      "Software packages supported by the Qiskit community, not maintained by IBM Quantum.",
+    icon: "group.svg",
+  },
+];
+
+const EcosystemJoinLink: TextLink = {
```

Comment:
I think this name should be camel cased:

```suggestion
const ecosystemJoinLink: TextLink = {
```